wfl/Models/Restaurant.cs

```diff
@@ -1,37 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Web;
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.IdGenerators;
+using MongoDB.Bson.Serialization.Attributes;
 
 namespace wfl.Models
 {
     public class Restaurant
     {
-        public static IList<Restaurant> Restaurants;
-        private static int greatestID;
-        private string name;
-        private int id;
-
-        static Restaurant()
-        {
-            Restaurants = new List<Restaurant>();
-            greatestID = -1;
-        }
-
-        public string Name {
-            get { return name; }
-            set { name = value; }
-        }
-
-        public int ID {
-            get { return id; }
-            set { id = value; }
-        }
-
-        public static int NextID()
-        {
-            greatestID += 1;
-            return greatestID;
-        }
+        [BsonId(IdGenerator = typeof(ObjectIdGenerator))]
```

Comment:
Referencing a MongoDB class in your model objects ties the app to only using MongoDB as a persistence engine. Ideally you want to encapsulate code that is database implementation specific into one (or two) classes. This allows you to do things later like switch from using MongoDB to cassandra or dynamoDB etc... without having to refactor all of your code.   This will likely require you to change how your DBManager code works. For example maybe the call to create a new restaurant automatically assigns the Id property of the restaurant.